autopush/main.py

```diff
@@ -134,6 +134,10 @@ def add_shared_args(parser):
     parser.add_argument('--router_write_throughput',
                         help="DynamoDB router write throughput",
                         type=int, default=5, env_var="ROUTER_WRITE_THROUGHPUT")
+    parser.add_argument('--router_timeout',
+                        help="Seconds to wait for a router to process a "
+                             "message",
```

Comment:
No, it's timing out the whole router transaction. The `connectTimeout` parameter limits how long it takes to connect to a peer, where this would limit the entire transaction time. Which, I believed, was the problem. If not, then I'm not sure I understand what the issue was.